notes/reflection/ReflectionWellScoped.agda

```diff
@@ -1035,6 +1048,134 @@ mkMacro f hole = R.bindTC R.getContext λ ctx →
       where nothing → mkTC (R.typeError (R.strErr "The IMPOSSIBLE has happened" ∷ []))
     f t)
 
+record Kit (◆ : Nat → Set) : Set where
+  field
+    reify : ◆ m → List (Arg (Term m)) → Term m
+    thin  : Thin m n → ◆ m → ◆ n
+    var₀  : ◆ (suc m)
+
```

Comment:
While `Kit` is ok, I find the names `◆`, `KIT`, `Kittable`, and `kitX` a bit non-telling.  How about drawing names from _replacement_?  I suggest the following name swaps:
- `◆ --> Rep` (alluding to "Replaceable" or "Representation")
- `KIT --> Replace`
- `Kittable --> Replaceable`
- `kitX --> replaceX`